### PR TITLE
Contact me section

### PR DESCRIPTION
**Pull request by abdulrahmanshr75**
- added the contact me section
- added the form
- limited the name and text area
- implemented the button interactions
- added the illustration

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -150,65 +152,134 @@ <h2>Multi-Post Stories Gain+Glory</h2>
...
+            placeholder="Email Address"
+            required
+          />
+          <textarea
+            class="Form-text"
+            name="text"
+            maxlength="500"
+            placeholder="Enter Text Here"
+            required
+          >
```
Comment (andersonlebon) [inline index.html]: - [ ] Kindly remove this unnecessary character `>`

File: index.html [hunk 2]
```diff
@@ -150,65 +152,134 @@ <h2>Multi-Post Stories Gain+Glory</h2>
...
+            type="email"
+            placeholder="Email Address"
+            required
+          />
+          <textarea
+            class="Form-text"
+            name="text"
+            maxlength="500"
+            placeholder="Enter Text Here"
+            required
```
Comment (andersonlebon) [inline index.html]: - [ ] One of the requirements says that you should stick to the design templates in Figma as much as possible, this includes the font, colors, images, and spacing (margin/padding). Kindly let's go ahead a  make our form look exactly like the template one. Refer yourself to these screenshots below  :point_down: :point_down: 

 Your design | The template
-----------------|------------------
Screenshot from 2021-12-07 18-44-10 | Screenshot from 2021-12-07 18-40-54

**Review by andersonlebon — CHANGES_REQUESTED** [review]
### Hi @abdulrahmanshr75  

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by alonsofalconi — APPROVED** [review]
Hi @abdulrahmanshr75 and @aeon9300,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**